src/bundle/Resources/public/scss/ui/modules/universal-discovery/_grid.item.scss

```diff
@@ -25,9 +25,15 @@
         background: $ibexa-white;
     }
 
+    &__name-text {
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
```

Comment:
In the previous design, it was ellipsis after second line not in the first line.